src/platform/brand-consolidation/feature-flag.js

```diff
@@ -0,0 +1,8 @@
+/**
+ * Returns whether the current Webpack build is executing as the brand-consolidation of Vets.gov/VA.gov.
+ * @returns {boolean}
+ * @module platform/brand-consolidation/feature-flag
+ */
+export default function isBrandConsolidationEnabled() {
+  return window.settings.brandConsolidationEnabled;
+}
```

Comment:
@U-DON This is the module/feature flag for checking brand-consolidation builds. I'm hoping to have it integrated throughout in the near future.